docs/5.x/cluster.md

```diff
@@ -2063,58 +2063,39 @@ With `promiscuous-bridge`, the behavior is similar to that of the kubenet networ
     see [Kernel Modules](/requirements/#kernel-modules) for details.
 
 
+### WireGuard Encrypted Networking
+
+Gravity supports encrypting all pod-to-pod traffic between hosts using [WireGuard](https://www.wireguard.com) to create
+an encrypted VPN between hosts. This feature is configured through the application manifest when building gravity 
+applications:
+
+```yaml
+providers:
+  generic:
+    network:
+      type: wireguard
+```
+
+!!! tip "Version"
+    The WireGuard feature is only available from 5.5.0-alpha.1 and later
+
+!!! tip "Kernel module":
+    The WireGuard feature currently requires the WireGuard kernel module to be installed and available on the host. Please see
+    the [WiregGuard installation instructions](https://www.wireguard.com/install/) for more information.
+
+## Customizing Cluster DNS
+
+Gravity uses [CoreDNS](https://coredns.io) for DNS resolution and service discovery within the cluster.
+
+The coredns configuration can be edited after installation, by updating the `kube-system/coredns` configmap.
+
 
 [//]: # (Footnotes and references)
 
 [PodSecurityPolicies]: https://kubernetes.io/docs/concepts/policy/pod-security-policy/
 [RBAC]: https://kubernetes.io/docs/admin/authorization/rbac/
 [promiscuous-mode]: https://en.wikipedia.org/wiki/Promiscuous_mode
 
-## Customizing Cluster DNS
 
-Gravity uses [CoreDNS](https://coredns.io) for DNS resolution and service discovery within the cluster.
 
-The following configuration is used by default, which can be customized and applied to a gravity cluster. The default
-configuration will be restored automatically if the `kube-system/coredns` configmap is deleted from kubernetes.
 
-```
```

Comment:
I wonder why you're removing the docs on the structure of the configmap for coredns?

Reply:
This was changed in the release. The configmap is no longer processed by planet, and no longer runs through go template when changed.